system-test/bigquery.ts

```diff
@@ -101,11 +101,13 @@ describe('BigQuery', () => {
   ];
 
   before(async () => {
-    // Remove buckets created for the tests.
-    await deleteBuckets();
+    await Promise.all([
```

Comment:
I'm surprised this would cause flakiness, the difference between these two approaches would be that when you go:

```js
await deleteBuckets();
await deleteDatasets();
```

delete buckets always happens before delete datasets.

Whereas, with:

```js
Promise.all([
  deleteBuckets(),
  deleteDatasets()
])
```

It's going to try to delete in parallel, scratching my head 🤔